README.md

```diff
@@ -202,6 +202,9 @@ Create a `.env` file containing the following variables:
 
     ![OAuth token](docs/assets/slack/oauth-token.png)
 
+- `EMAIL_PATTERNS`
+  Email patterns of your organization (separated by comma)
```

Comment:
Can you please provide more detail as to when this comes into play?

Reply:
What kind of info you want to see here please? I setup it as mandatory environment variable as this bot can be used by someone outside MM organization

Reply:
Something along the lines of: "used to find slack users based on gitlab IDs", but better. It should answer the question: "OK cool, it allows to find slack users based on gitlab IDs, but to do what exactly? Notifying commit authors on slack? Notifying Gitlab reviewers on Slack?".

Reply:
Ok I understood and will provide more details based on what you said

Reply:
```suggestion
  Email patterns of your organization (separated by comma),
  for instance `@my.org,@contractors.my.org`.
  Used to find Slack users from Gitlab IDs.
  Note that this means your org has to use the same naming scheme for emails and gitlab IDs.
```